go-controller/pkg/informer/informer_test.go

```diff
@@ -87,11 +102,12 @@ var _ = Describe("Informer Event Handler Tests", func() {
 			ReceiveAllUpdates,
 		)
 
-		stopChan := make(chan struct{})
-		defer close(stopChan)
-
 		f.Start(stopChan)
-		go e.Run(1, stopChan)
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			e.Run(1, stopChan)
+		}()
```

Comment:
you really ought to be able to say `wg.Go(func() { e.Run(1, stopChan) })`...
(ah, but it's already been proposed and rejected)

Reply:
Yeah :(